widgets/heatmap/internal/axes/label.go

```diff
@@ -60,5 +126,13 @@ func xLabels(yEnd image.Point, graphWidth int, labels []string, cellWidth int) (
 // the X label belongs to the middle column of the three columns,
 // and the padded length is 3*3 (cellWidth multiplies the number of columns), which is 9.
 func paddedLabelLength(graphWidth, longest, cellWidth int) (l, index int) {
```

Comment:
Considering that the scope of the `l` variable here is the entire function and it is also a named return parameter, can we give it a bit more readable name? How about 'length` instead?